ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/core/TransactionPool.java

```diff
@@ -131,6 +134,13 @@ public PendingTransactions getPendingTransactions() {
       return basicValidationResult;
     }
 
+    String sender = transaction.getSender().toString();
+    if (!checkIfAccountIsWhitelisted(sender)) {
```

Comment:
Now that I read this, I think the if statement will be better as something like:

```
if (!accountIsWhitelisted(sender)) {
```

What do you think?

Reply:
Or maybe even `accountIsNotWhitelisted(sender))` so we don't need the negative... idk

Reply:
actioned